sql/json-functions-generated-column.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Using TiDB as a Document Store
+category: user guide
+---
+
+# Using TiDB as a Document Store
+
+## About
+
+In order to functionally keep pace with the latest version of MySQL (version 5.7 and above) and provide better support for document storage business as well, TiDB supports JSON in the latest version. You can use JSON type fields in TiDB tables, and index the fields in the JSON document in the form of generated column. Based on this, you can flexibly deal with those business with uncertain schema, no longer limited by the bad read performance and lack of support for transaction in traditional document databases.
+
+## JSON Functions
+
+The support for JSON in TiDB mainly refers to the user interface of MySQL 5.7. For example, you can create a table that includes a JSON field to store complex information:
+
+```sql
+CREATE TABLE person （
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    address_info JSON
+）
+```
+
+When you insert data into a table, you can deal with those data with uncertain schema like this:
+
+```sql
+INSERT INTO person (name, address_info) VALUES ("John", '{"city": "Beijing"}');
+```
+
+Put a legal JSON string at the location corresponding to the JSON field, and then you can insert JSON into the table. TiDB will parse the text and saves it in a more compact and easy-to-access form.
```

Comment:
You can insert JSON data into the table by inserting a legal JSON string into the column corresponding to the JSON field. TiDB will then parse the text and save it in a more compact and easy-to-access binary form.